recipes/msys2/all/conanfile.py

```diff
@@ -68,9 +63,16 @@ def validate(self):
         if self.settings.arch != "x86_64":
             raise ConanInvalidConfiguration("Only Windows x64 supported")
 
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
```

Comment:
It's useless and confusing. There is no compiler in settings.

Reply:
Hi @SpaceIm,
I guess I am the one who is confused. Please see https://github.com/conan-io/hooks/issues/471